manifest.json

```diff
@@ -25,6 +25,11 @@
       "matches": ["https://scratch.mit.edu/*"],
       "run_at": "document_start",
       "js": ["content-scripts/prototype-handler.js"]
+    },
+    {
+      "matches": ["https://scratch.mit.edu/*"],
+      "run_at": "document_start",
+      "js": ["content-scripts/load-redux.js"]
```

Comment:
The same match and run at are shared between this 2 JS files, please merge them into a single object with two files in the "js" array.